src/App.js

```diff
@@ -129,6 +171,28 @@ class App extends Component {
           <div className="Title">
             { calculatorInfo && calculatorInfo.title}
           </div>
+          <div className="Content Stack">
+            <div className="Control">
+              <input 
+                type="text" 
+                value={this.state.newExclusionReason}
+                onChange={this.handleReasonChange}
+                placeholder="Enter a date exclusion reason">
+              </input>
+              &nbsp;&nbsp;
```

Comment:
This is being used to get the spacing between the `input` and the `DatePicker`?

Maybe this is an opportunity to give it a className with some styles or to leverage some `flexbox` property...

Reply:
Yeah, sorry, I'm really bad at CSS!